coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/TrinoSqlUDFConverter.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2017-2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino;
+
+import java.util.Map;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.util.SqlShuttle;
+
+import static com.linkedin.coral.trino.rel2trino.CoralTrinoConfigKeys.*;
+
+
+/**
+ * This class extends the class of SqlShuttle and calls CalciteTrinoUDFOperatorTransformers to traverse
+ * the hierarchy and converts UDF operator in all SqlCalls if it is required
+ */
+public class TrinoSqlUDFConverter extends SqlShuttle {
```

Comment:
@aastha25 should this be merged with/cascaded with an existing Shuttle?

Reply:
I would vote for merging it in `CoralSqlNodeToTrinoSqlNodeConverter`, we may even consider migrating existing `CoralSqlNodeToTrinoSqlNodeConverter` to use `SqlCallTransformer`.

Reply:
Talked with @ljfgem offline about `CoralSqlNodeToTrinoSqlNodeConverter`. As it is in a pending PR #315, we will explore how to merge the functionality of this class with `CoralSqlNodeToTrinoSqlNodeConverter` later when PR #315 is merged.

Reply:
I rename it as `CoralToTrinoSqlCallConverter` for now. So there will not be any code conflict when #315 is merged. We can merge the code from `CoralToTrinoSqlCallConverter` into `CoralSqlNodeToTrinoSqlNodeConverter` and remove the class of `CoralToTrinoSqlCallConverter` if we decide to merge them.